solr/core/src/test/org/apache/solr/handler/admin/api/UnloadCoreAPITest.java

```diff
@@ -56,7 +56,6 @@ public void setUp() throws Exception {
   public void testValidUnloadCoreAPIResponse() throws Exception {
     SolrJerseyResponse response = unloadCoreAPI.unloadCore(coreName, getUnloadCoreRequestBodyObj());
     assertEquals(0, response.responseHeader.status);
-    assertNotNull(response.responseHeader.qTime);
```

Comment:
Why did this test change?

Reply:
According to error-prone:

> error: [ImpossibleNullComparison] This value cannot be null, and comparing it to null may be misleading.

This is because primitive data types have always a default value. Resolving this to focus on the root discussion.

Reply:
We need to be careful with errorprone updates. See https://github.com/apache/solr/blob/main/gradle/validation/error-prone.gradle#L68 we want to make sure we list and explicitly enable/disable checks. Just upgrading means we aren't going to handle all the checks we may not want to run since they are either covered by other tools (ie: forbidden apis) or they are not necessary for our codebase.